psalm.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="true"
```

Comment:
We should probably use `errorLevel="1"` instead

Reply:
Updated that parameter now as suggested.